activiti-image-generator/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
@Salaboy Really? You are also committed this file to other projects?
https://github.com/Activiti/Activiti/commits/master/activiti-engine/.gitignore